templates/api.hbs

```diff
@@ -1,17 +1,24 @@
 {{>header}}
 
-import { Express } from 'express'
+import { Express, Request, Response } from 'express'
 import passport from 'passport'
 import * as t from './types'
 import * as v from '../../validation'
+{{#if (isAnyOperationSupportMultipart operations)}}
+import * as f from '../../impl/helpers/{{lowerCase name}}MultipartHelper'
```

Comment:
Should be `{{identifier name}}` I think, so it matches the way the file name is generated.